### DISCUSSION TITLE
when i rebuild source code,how to ignore the part of presentationeditor、spreadsheeteditor and the uselessful platform for me like embed and mobile ?

### DISCUSSION THREAD

**Discussion by alfabeto2019** [ONLYOFFICE/build_tools#352] [Q&A] [answered]
Do we support this feature when rebuilding?

**Comment by ShockwaveNN** [comment]
Moved this issue to discussion, since it's not an issue, but a question

About question itself - I'm not 100% sure but I think we have no such option

But generally speaking builing web interface took around 2-5 minutes, most of compilation process are too by `core` libraries, which almost 100% cannot be skipped, since they shared for all editors

**Comment by trofim24** [comment, marked as answer]
@alfabeto2019 
Sorry, we have no such option
